ports/hffix/CMakeLists.txt

```
@@ -0,0 +1,28 @@
+cmake_minimum_required(VERSION 3.10)
+
+set(PROJECT_NAME hffix)
+project(${PROJECT_NAME})
+
+include(GNUInstallDirs)
+
+add_library(hffix-header INTERFACE)
```

```suggestion
add_library(hffix INTERFACE)
```
In my opinion, it might be better to rename it as `hffix`.